boto/glacier/concurrent.py

```diff
@@ -237,7 +237,7 @@ def _process_chunk(self, work):
             except self._retry_exceptions as e:
                 log.error("Exception caught uploading part number %s for "
                           "vault %s, attempt: (%s / %s), filename: %s, "
-                          "exception: %s, msg: %s",
+                          "exception: %s as msg: %s",
```

Comment:
d'oh -- sorry about this.  I did review the changes but managed to miss this one :-/